workloads/kube-burner/common.sh

```diff
@@ -195,3 +174,37 @@ remove_update_taint() {
   # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
   oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
 }
+
+
+label_node_with_label() {
+  colon_param=$(log $1 | tr "=" ":" | sed 's/:/: /g')
+  export POD_NODE_SELECTOR="{$colon_param}"
+  export NODE_SELECTOR="$1"
+  NODE_COUNT=$(oc get node -o name --no-headers -l node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",node-role.kubernetes.io/worker= | wc -l )
+  if [[ ${NODE_COUNT} -le 0 ]]; then
+    log "Node count <= 0: ${NODE_COUNT}"
+    exit 1
+  fi
+  export WORKER_NODE_NAMES=$(oc get node -o name --no-headers -l node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",node-role.kubernetes.io/worker= | head -n ${NODE_COUNT})
```

Comment:
we don't need to export this variable

Reply:
@rsevilla87 if i don't need to export, should I put to variable to lower case?